docs/src/index.md

```diff
@@ -0,0 +1,16 @@
+# DataJoint Element - Lab
+
++ `element-lab` features a DataJoint pipeline design for lab related information, such as Lab, User, Project, Protocol, Source.
+
++ `element-lab` is not a complete workflow by itself, but rather a modular design of tables and dependencies. 
+
++ `element-lab` can be flexibly attached to any DataJoint workflow.
+
++ See the [Element Lab documentation](https://elements.datajoint.org/description/lab/) for the background information and development timeline.
+
++ For more information on the DataJoint Elements project, please visit https://elements.datajoint.org.  This work is supported by the National Institutes of Health.
+
+## Element architecture
+
+![element lab diagram](https://raw.githubusercontent.com/datajoint/element-lab/d222f673e590979a92ff815adb880f474eed338e/images/lab_diagram.svg)
```

Comment:
Please move this to concepts following the [dlc site](https://datajoint.com/docs/elements/element-deeplabcut/0.2/concepts/) as a model